### ISSUE TITLE
Disable extension notifications / popups

### ISSUE THREAD

**Issue by evertjr** [continuedev/continue#2437]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://docs.continue.dev/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS: macOS
- Continue: v0.9.213 (pre-release)
- IDE: vscode
- Model:
- config.json:
```

### Description

For many reasons this extensions triggers popups in vscode and it does not respect our notifications preference so they keep appearing without consent.

This is especially distracting for notifications such as this:

```
Warning: qwen2.5-coder:1.5b-instruct-q8_0 is not trained for tab-autocomplete, and will result in low-quality suggestions. See the docs to learn more about why: https://docs.continue.dev/features/tab-autocomplete#i-want-better-completions-should-i-use-gpt-4
```

it's okay to show this warning once but not every single time, specially since this is a false positive, the model works fine.

### To reproduce

1. check "turn off all notifications from Continue..."
2. still get notifications from continue

**Comment by Patrick-Erichsen** [comment]
Hi @evertjr , thanks for the heads up here. Could you share where you have configured notifications to be disabled? I'm not seeing much in the VS Code docs around how to check this setting so that we can then respect it.

**Comment by evertjr** [comment]
e believe it's from the native Notification Center, this notification shows up even if I enable do not disturb.

<img width="620" alt="image" src="https://github.com/user-attachments/assets/13b0c533-7b32-420b-9761-de82246850a9">

**Comment by RomneyDa** [comment]
Hi @evertjr, 

Would agree that it should not be throwing an error; we can change this to a warning message, since error messages will be shown regardless of the info/warning settings to make sure real errors aren't silent. Any other specific errors notifications that have been showing incessantly?

Notifications playground: clone and debug this repo: [Sample notifications extension](https://github.com/microsoft/vscode-extension-samples/tree/main/notifications-sample)

**Comment by RomneyDa** [comment]
note to self, error messages are shown by:
- uncaught error in core, ide, or gui
- ide.showToast("error" | "warning" | "info", ...) from core
- 
Quick scan through the ~200 Error throws and show, all look like valid error events except this one and one other. Will open PR

**Comment by RomneyDa** [comment]
PR opened
https://github.com/continuedev/continue/pull/2619

**Comment by AnnoyingTechnology** [comment]
btw, the error being triggered is in itself is a problem. 

Qwen 2.5 **has** been trained for FIM. It would be nice to remove this warning altogether.

![image](https://github.com/user-attachments/assets/29dd744b-5cc7-46ce-aaad-adc58c50bffa)

**Comment by Patrick-Erichsen** [comment]
@AnnoyingTechnology yes, but that is the fine-tuned `instruct` model which is not trained for FIM.
